<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuwidget</class>
 <widget class="QWidget" name="menuwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="calButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>指数运算请不要输入较大数字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightlineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>指数运算请不要输入过大数字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="duquButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>读取文件</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>^</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="anslineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>答案</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>答案位数</string>
   </property>
  </widget>
  <widget class="QLabel" name="answeilabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>计算时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="timelabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>440</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cunButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>答案存文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>读取文件的路径在目录data文件夹的cal.txt文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>存取文件的路径在目录data文件夹的anc.txt文件</string>
   </property>
  </widget>
  <widget class="QComboBox" name="jcomboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>二进制</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>十进制</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
